src/libraries/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/ManagedTextSection.cs

```diff
@@ -40,8 +41,8 @@ internal sealed class ManagedTextSection
         public int MetadataSize { get; }
 
         /// <summary>
-        /// The size of managed resource data stream.
-        /// Aligned to <see cref="ManagedResourcesDataAlignment"/>.
+        /// The size of managed resource data stream (unaligned).
```

Comment:
@tmat Could you please take a look at the changes in this file? How are the alignment requirements expected to be handled by `ManagedPEBuilder`?